views/sms/index.js

```diff
@@ -241,7 +241,7 @@ exports.respond = function(req, res) {
             return save_utils.countRequestsPerRegion(region_code);
         }).then( function(numRequests) {
             requestData = save_utils.createSerial(numRequests, req.cookies.request_object, region_code);
-            requestData.is_sms = 'sms';
+            requestData.origin = 'sms';
```

Comment:
Well, the "source" is the user-visible name and has always had that name.  I renamed the "is_sms" variable to "origin," as mentioned in the PR message, because that variable isn't just about whether or not the request came via SMS anymore.

I can make everything be `source`, sure.